README.md

```diff
@@ -4,23 +4,50 @@ This project is a port of the ownCloud documentation, that was previously genera
+2. In a feature branch, branched from master, change the relevant [AsciiDoc](./docs/what-is-asciidoc.md) files.
+3. Push your feature branch to the repository and create a PR from it.
+4. Make any requested changes.
+5. Your PR will be merged.
+
 ## Contributing to the Docs
 
-If you've been contributing to the previous version of ownCloud's documentation, which used reStructuredText and Sphinx-Doc, here’s how to get started contributing to the new version of the documentation.
+If you've been contributing to the previous version of ownCloud's documentation, which used reStructuredText and Sphinx-Doc, here's how to get started contributing to the new version of the documentation.
+Unlike the previous version of the documentation, you don't, necessarily, need to install a large number of tools and packages just to review your changes.
+
+### Only Making Text Changes?
+
+If you're only making text changes, then install either:
+
+1. The [AsciiDoc Live Preview plugin](https://asciidoctor.org/docs/editing-asciidoc-with-live-preview/) for *Firefox*, *Google Chrome*, or *Opera*
+2. The [AsciiDoc Live Preview plugin](https://asciidoctor.org/docs/editing-asciidoc-with-live-preview/#using-a-modern-text-editoride) for your text editor or IDE, *if it has one*.
+
+Using one, or both, of these, you can quickly check if the changes you make are what you expect, and if there are any render errors.
+If the document renders as you expect, then you can commit the changes and push them to the docs repository.
+
+### Are You Working With Inline Code Examples, Images, and Attachments?
+
+If, however, you're working with inline code examples, images, and attachments, then you need to [install Antora's command-line tools](./docs/install-antora.md).
+After that, you need to learn how to [build the docs](./docs/build-the-docs.md) from the command line and to how to review the changes in your browser.
+
+This is because if you use a Live Preview plugin, it won't render the links to external files correctly, as it won't have the correct path information.
+In this case, you need to use Antora to regenerate the documentation, and see if it reports any errors, or if something looks amiss when reviewing the changes in your browser.
 
-You need to do a few things to contribute to the documentation:
+### Now You're Ready to Contribute
 
-1. [Install Antora](./docs/install-antora.md).
-2. Learn [the AsciiDoc file format's basics](./docs/what-is-asciidoc.md).
-3. Update your toolset so that it supports both AsciiDoc and Antora.
-4. Learn how to [build the docs](./docs/build-the-docs.md).
-5. Create meaningful commits and a PR with your changes.
+Now that you've installed the required tools, you're ready to contribute.
+So dive on in and learn [the AsciiDoc basics](./docs/what-is-asciidoc.md).
```

Comment:
dive on in?

Reply:
Perhaps slang that only I understand. Thanks for pointing that out.